migration/2.3/migrationScripts/migrate.js

```diff
@@ -0,0 +1,142 @@
+const async = require('async');
+const glob_entries = require('webpack-glob-entries');
+const shelljs = require('shelljs');
+const path = require('path');
+const fs = require('fs');
+const { copy, log, PROJECT_ROOT } = require("./common");
+log("Migrating your project to finsemble-seed v2.3");
+
+//
+
+function copyFoldersAndFiles(cb) {
+    async.eachSeries([
+        {
+            oldPath: path.join(PROJECT_ROOT, "build"),
+            newPath: path.join(PROJECT_ROOT, "build-original"),
+            messageOnComplete: "Copied old build directory to build-original.",
+            deleteAfterCopy: true
+        },
+        {
+            oldPath: path.join(__dirname, "../build"),
+            newPath: path.join(PROJECT_ROOT, "build"),
+            messageOnComplete: "Copied new build directory into the project."
+        },
+        {
+            oldPath: path.join(PROJECT_ROOT, "server"),
+            newPath: path.join(PROJECT_ROOT, "server-original"),
+            messageOnComplete: "Copied old server directory to server-original.",
+            deleteAfterCopy: true
+        },
+        {
+            oldPath: path.join(__dirname, "../server"),
+            newPath: path.join(PROJECT_ROOT, "server"),
+            messageOnComplete: "Copied new server directory into the project."
+        },
+        {
+            oldPath: path.join(__dirname, "../src-built-in"),
+            newPath: path.join(PROJECT_ROOT, "src-built-in"),
+            messageOnComplete: "Copied new presentation components into src-built-in."
+        },
+        {
+            oldPath: path.join(PROJECT_ROOT, "gulpfile.js"),
+            newPath: path.join(PROJECT_ROOT, "gulpfile.js.original"),
+            messageOnComplete: "Copied old gulpfile.",
+            deleteAfterCopy: true
+        },
+        {
+            oldPath: path.join(__dirname, "../gulpfile.js"),
+            newPath: path.join(PROJECT_ROOT, "gulpfile.js"),
+            messageOnComplete: "Copied new gulpfile.",
+        },
+        {
+            oldPath: path.join(PROJECT_ROOT, "build-original/webpack/webpack.files.entries.json"),
```

Comment:
@carterbs what about the new webpack.files.entries.default.json? I guess the customer will just have duplicates of those components? What would be the right way for a customer to *remove* a component that they don't want if it's in the default.json? I'm not sure what the right answer is for that one.

+ @NinjaManatee